config/core/configmaps/autoscaler.yaml

```diff
@@ -59,6 +59,22 @@ data:
     # NOTE: Only one metric can be used for autoscaling a Revision.
     container-concurrency-target-default: "100"
 
+    # The container concurrency max limit is an operator setting ensuring that
+    # the individual revisions cannot have arbitrary large concurrency
+    # values, or autoscaling targets.
```

Comment:
🤔 Do you plan to reject Service/Configuration updates with values higher than this?  How will this be enforced?

Reply:
My idea was to use validating webhooks. We pass config maps there now, so it should be possible 🙄

Reply:
Part of the reason I ask is that this seems like the wrong configmap to put this in if that's your goal.  Let's find time tomorrow to chat?  `config-defaults` is what I'd expect.

Reply:
SG.